service/src/main/kotlin/fi/espoo/evaka/process/ProcessMetadataController.kt

```diff
@@ -259,4 +324,87 @@ class ProcessMetadataController(private val accessControl: AccessControl) {
                 )
             }
             .exactlyOne()
+
+    private fun Database.Read.getApplicationDocumentMetadata(
+        applicationId: ApplicationId
+    ): DocumentMetadata =
+        createQuery {
+                sql(
+                    """
+        SELECT 
+            CASE 
+                WHEN a.type = 'DAYCARE'
+                THEN 'Varhaiskasvatus- ja palvelusetelihakemus'
+                WHEN a.type = 'PRESCHOOL'
+                THEN 'Ilmoittautuminen esiopetukseen ja / tai valmistavaan opetukseen'
+                WHEN a.type = 'CLUB'
+                THEN 'Kerhohakemus'
+            END AS name,
+            coalesce(a.sentdate, a.created) AS created_at,
```

Comment:
This uses DB connection time zone for date -> datetime cast, and should probably use Europe/Helsinki explicitly